lib/HTFeed/Stage/ImageRemediate.pm

```diff
@@ -557,30 +556,7 @@ sub _remediate_jpeg2000 {
         }
     }
 
-    # Use kdu_munge to strip out the existing XMP. This may help in situations
-    # where exiftool cannot update existing XMP because it is in a "huge
-    # JPEG2000 box". Note that as of exiftool 10.03, exiftool should be able to
-    # handle "extended-size JPEG2000 boxes" so long as they are <4GB.
-    #
-    # kdu_munge is a modification to kdu_transcode; see
-    # http://websites.umich.edu/~roger/kdu_transcode.html
-    my $kdu_munge = get_config('kdu_munge');
-    if (    $self->{volume}->get_nspkg()->get('use_kdu_munge')
-        and defined $kdu_munge
-        and $kdu_munge )
-    {
-        system("$kdu_munge -i '$infile' -o '$outfile' > /dev/null 2>&1")
-          and $self->set_error(
-            "OperationFailed",
-            file      => $outfile,
-            operation => "kdu_munge"
-          );
-
-        $self->update_tags( $exifTool, $outfile );
-    }
-    else {
-        $self->update_tags( $exifTool, $outfile, $infile );
-    }
+    $self->update_tags( $exifTool, $outfile, $infile );
```

Comment:
Does grk_compress not copy over the XMP metadata? Or does it avoid the "large block" problem?

Reply:
exiftool as of 10.03 can handle the "large blocks".

Reply:
It would probably be worth a test case with a 'huge JPEG2000 box' to ensure we end up with something valid.

Reply:
I wasn't able to produce a small test file with a huge JPEG2000 box, but I did manually test with the item you provided & ensure that the output image still contained the original XMP fields. I also added a test around copying over XMP; by default, I found that grok doesn't preserve the XMP in all situations, and that `ExifTool::SetNewValuesFromFile` needs an extra option to ensure it preserves the group of the tag (i.e. copies XMP tags to XMP instead of to IFD0 etc)